aws_marketplace/using_model_packages/preprocessing-audio-data-using-a-machine-learning-model/preprocessing-audio-data-using-a-machine-learning-model.ipynb

```diff
@@ -0,0 +1,894 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #20.        file_id = row['file_path'].split('/')[-1]#.split('.')[0]
</pre><p>Clean up commented end of this line?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/3061/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='49'/>